site/content/docs/5.0/components/alerts.md

```diff
@@ -74,6 +74,62 @@ Alerts can also contain additional HTML elements like headings, paragraphs and d
 </div>
 {{< /example >}}
 
+### Icons
```

Comment:
I think the whole section should be dropped since Boosted already includes icons in alerts and certainly won't use Bootstrap Icons.